src/go/k8s/tests/e2e/admin-api-tls-ext/01-redpanda-cluster.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: cluster-admin-api-tls-ext
+spec:
+  image: "localhost/redpanda"
+  version: "dev"
+  replicas: 1
+  resources:
```

Comment:
Do we need to be specifying resources this large for tests?

Reply:
I think there's a balance between making Redpanda start quickly and not overflowing the CI, but the second aspect controlled by the concurrency level set on the E2E tests (i.e. I took it from similar tests)